server/src/main/java/org/apache/druid/server/coordinator/CuratorLoadQueuePeon.java

```diff
@@ -335,8 +335,6 @@ public void stop()
 
     queuedSize.set(0L);
     failedAssignCount.set(0);
-    processingExecutor.shutdown();
```

Comment:
Should we shut these down somewhere? (Maybe tie `LoadQueueTaskMaster` to lifecycle and shutdown the execs there?)

Reply:
That seems reasonable, :+1:

Reply:
Used the `factory` passed into the method in `CliCoordinator` to create the executors, which itself appears to be under lifecycle.

Reply:
Ah, that won't work...

Reply:
>Ah, that won't work...

Fixed to use `ExecutorServices.manageLifecycle` to tie the executors to the main service lifecycle.